docs/PLAN_WALLET_KIT.md

```
@@ -0,0 +1,152 @@
+## 3. Pasos del plan (no ejecutar hasta aprobar)
+
+### Fase 1: Instalación y revisión del block
+
+0. **Inicialización obligatoria (requerido por la CLI)**
+   - Antes de añadir cualquier block, hay que ejecutar **una vez**:
+     ```bash
+     npx trustless-work init
+     ```
+   - **Importante:** Usar `npx trustless-work init` (no `trustless-work init`); sin `npx` el comando no se encuentra.
+   - Este paso instala dependencias y crea `.twblocks.json` (configuración uiBase). Si no se hace, al ejecutar `npx trustless-work add wallet-kit` aparece:
+     ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)